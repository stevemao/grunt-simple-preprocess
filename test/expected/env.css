body {
  background: yellow;
}
body {
  background: green;
}
