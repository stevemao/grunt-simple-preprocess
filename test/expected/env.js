var value = 'test';
var value = 'test and prod';
