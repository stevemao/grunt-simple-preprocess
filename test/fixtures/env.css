/* @if dev */
body {
  background: red;
}
/* @endif */

/* @if test*/
/*body {
  background: yellow;
}*/
/* @endif*/

/* @if prod */
/* body {
  background: blue;
}*/
/* @endif */

/* @if test,prod */
/*body {
  background: green;
} */
/* @endif */
