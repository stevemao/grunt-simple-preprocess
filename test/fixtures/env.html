<!-- @if dev -->
<script charset="utf-8">
  var value = 'dev';
</script>
<!-- @endif -->

<!-- @if test -->
<!--<script charset="utf-8">
  var value = 'test';
</script> -->
<!-- @endif -->

<!-- @if prod -->
<!--<script charset="utf-8">
  var value = 'prod';
</script>-->
<!-- @endif -->

<!-- @if test, prod -->
<!--<script charset="utf-8">
  var value = 'test and prod';
</script>-->
<!-- @endif -->
