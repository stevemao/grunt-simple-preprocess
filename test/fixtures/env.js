// @if dev
var value = 'dev';
// @endif

// @if test
// var value = 'test';
// @endif

// @if prod
// var value = 'prod';
// @endif

/* @if test, prod */
/*var value = 'test and prod';*/
/* @endif */
