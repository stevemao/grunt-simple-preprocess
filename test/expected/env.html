<script charset="utf-8">
  var value = 'test';
</script>
<script charset="utf-8">
  var value = 'test and prod';
</script>
